programs/gateway/src/state/registrar.rs

```diff
@@ -25,6 +25,10 @@ pub struct Registrar {
     /// The Gatekeeper Network represents the "Pass Type" that a 
     /// user must present.
     pub gatekeeper_network: Pubkey,
+    
+    /// If the plugin is one in a sequence, this is the previous plugin
+    /// If set, then update_voter_weight_record will expect a voter_weight_record owned by this registrar
+    pub predecessor_plugin_registrar: Option<Pubkey>,
```

Comment:
Should it be `previous_voter_weight_plugin` to make it more explicit what plugin type we are expecting here?

Reply:
I renamed it - but kept the "registrar" to make it clear it should be the pubkey of a Registrar struct